frontend/src/metabase/query_builder/components/QuestionActionButtons.jsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { t } from "ttag";
+
+import Button from "metabase/components/Button";
+import Tooltip from "metabase/components/Tooltip";
+import { Container } from "./QuestionActionButtons.styled";
+
+export const EDIT_TESTID = "edit-details-button";
+export const ADD_TO_DASH_TESTID = "add-to-dashboard-button";
+export const MOVE_TESTID = "move-button";
+export const CLONE_TESTID = "clone-button";
+export const ARCHIVE_TESTID = "archive-button";
+
+export const EDIT_ACTION = "edit";
+export const ADD_TO_DASH_ACTION = "add-to-dashboard";
```

Comment:
Can we move these constants to some shared constants file use them in other places too?

<img width="268" alt="Screen Shot 2021-07-20 at 14 01 15" src="https://user-images.githubusercontent.com/14301985/126313305-047628af-d0a7-4e44-99f2-f9c9a22a8abd.png">

Reply:
Moved all magic strings found in `QuestionActionButtons` and `QueryModals` to `MODAL_TYPES` in `metabase/query_builder/constants.js`